Sources/Command/edit().swift

```diff
@@ -7,8 +7,15 @@ import Path
 public func edit(path: Path) throws -> Never {
 #if os(macOS)
     let input:Script.Input = .path(path)
-    let deps = try StreamReader(path: path).compactMap { try ImportSpecification(line: $0, from: input) }
-    let script = Script(for: .path(path), dependencies: deps)
+    let reader = try StreamReader(path: path)
+    var style: ExecutableTargetMainStyle = .topLevelCode
+    let deps = try reader.compactMap { line in
```

Comment:
Older versions of swift don't type check closures with multiple lines so you'll need to specify the type of `deps` here and in the editor().swift file

```suggestion
    let deps: [ImportSpecification] = try reader.compactMap { line in
```

Reply:
When you say older which version of Swift do you have in mind?

Reply:
Also in the editor().swift file